donor-portal/src/components/navbar/styles.css

```diff
@@ -136,13 +136,47 @@
 }
 
 .nav-link {
-  margin: 18px;
   color: white;
-  font-size: 14px;
+  padding: 0 15px;
+  font-size: 16px;
+  letter-spacing: 3px;
+  height: 100%;
```

Comment:
Could we remove this so that the border only covers the text line height?

Reply:
yh sure